libc/startup/linux/riscv/start.cpp

```diff
@@ -115,12 +115,7 @@ static void call_fini_array_callbacks() {
 } // namespace LIBC_NAMESPACE
 
 using LIBC_NAMESPACE::app;
-
-// TODO: Would be nice to use the aux entry structure from elf.h when available.
```

Comment:
This commit drops the TODO about elf.h.

I *think* these types should eventually be `Elf64_auxv_t`. Or `ElfW(auxv_t)`.  I think we should retain that TODO.

Reply:
fixed